cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -1,13 +1,32 @@
-from typing import Any, Dict, List
+import logging
+from typing import Any, Dict, List, cast
+
+import requests
+from decouple import config  # type: ignore
+
+logger = logging.getLogger(__name__)
 
 
 class RemoteSettings:
     recipes: List[Dict[str, Any]] = []
+    url: str = cast(str, config("REMOTE_SETTING_URL", default=""))
+
+    def get_recipes(self) -> List[Dict[str, Any]]:
+        return self.recipes
 
-    @classmethod
-    def get_recipes(cls) -> List[Dict[str, Any]]:
-        return cls.recipes
+    def update_recipes(self, new_recipes: List[Dict[str, Any]]) -> None:
+        self.recipes = new_recipes
 
-    @classmethod
-    def update_recipes(cls, new_recipes: List[Dict[str, Any]]) -> None:
-        cls.recipes = new_recipes
+    def fetch_recipes(self):
+        try:
+            response = requests.get(self.url)
+            if response.status_code == 200:
+                data = response.json().get("data", [])
```

Comment:
What happens if it's invalid json?

Reply:
the validation part will come in separate PR, I planned to break it down, first landing the functionality as suppose nothing can break, and remaining part everything can break :D